packages/fx-core/resource/package.nls.json

```diff
@@ -998,8 +998,11 @@
   "driver.teamsApp.summary.publishTeamsAppSuccess": "Teams app %s successfully published to the admin portal.",
   "driver.teamsApp.summary.copyAppPackageSuccess": "Teams app %s was successfully copied to %s.",
   "driver.teamsApp.summary.copyIconSuccess": "%s icon(s) were successfully updated under %s.",
+  "driver.teamsApp.summary.validate": "Teams Toolkit has checked all categories of validation rules:\n\nSummary:\n%s failed, %s passed.\n%s\n%s\n\nA completed log of validations can be found in %s",
```

Comment:
has checked all categories of validation rules -> has checked against all validation rules

A completed log -> A complete log